Gemfile-rails-dependencies

```diff
@@ -32,7 +32,7 @@ end
 # automatically included.  This issue was fixed on the Rails
 # side in Rails 7.0.1, but is not yet fixed in the Rails 6.1.x
 # branch.  Discussion can be found here - https://github.com/mikel/mail/pull/1439
-if RUBY_VERSION >= '3.1' && version.split(' ').last < '7.0'
```

Comment:
Maybe just do `version = ...` at the top and run the case statement of a local variable?

Reply:
We skip this for:
- pre 6.1 because it doesn't have Action Mailbox
- 7.0.1+ because it does the same (presumably) itself

The only `-stable` build left is `5-2-stable` that we'll drop.

In any case, this doesn't hurt on 3.1. And there's no other way to work around this for 6.1.

I don't really want to overcomplicate this.

Ideally, we could:
```diff
- version = ENV['RAILS_VERSION'] || (File.exist?(version_file) && File.read(version_file).chomp) || ''
+ version = ENV['RAILS_VERSION'] || (File.exist?(version_file) && File.read(version_file).chomp) || "~> 6.0.0"
```

and combine the last two branches.

This would allow to keep `&& version.split(' ').last < '7.0'`.
Is it worth it?

Reply:
Or we could return from the case statement a skip variable? Seems like skipping it on rails 7 is a good idea to ensure we don't change its behaviour?